### DISCUSSION TITLE
New firmware for RLC-523WA (Pet-Detection)

### DISCUSSION THREAD

**Discussion by combatlord** [AT0myks/reolink-fw-archive#19] [Firmware sharing]
Hello,

this is the latest FW for the RLC-523WA i have got from Support (03.06.2023).

The main Feature is that it has now Pet-Detection.
Pet-Detection is declared here as Beta-feature.

https://support-d.reolink.com/attachments/token/KCvCCwPiYAQo4QMF77Bx6LAO6/?name=IPC_523128M5MP.2217_23051710.RLC-523WA.OV05A10.5MP.WIFI1021.PTZ.REOLINK.pak

Model: RLC-523WA
Build No.: build 23051710
Hardware No.: IPC_523128M5MP
Config Version: v3.1.0.0
Firmware Version: v3.1.0.2217_23051710_v1.0.0.31
WEbGui: V1.0.284

Changes since FW: v3.1.0.1643_22122310_v1.0.0.30
Changes listed here are from WebGui
EDIT: (refers to WebGui)
---------------------------------------------------------------
- Add "Horizontal tracking Range" on the left side
- Add "Monitor Point" under PTZ
- Add "Return to Monitor Point" (auto/time) under PTZ
- Add "Interframe Space" under Basic Settings/Encode
- Remove "Switch to Flashplayer"
- Add "Reset All Settings" under Device Settings/Camera
- Add "Interframe Space" under Device Settings/Stream -> Clear/Fluent
- Add "Frame Rate Mode" Auto/Gradual/Constant under Device Settings/Stream
- Rename "Detection Alarm" to "Alarm Settings"
- Add "Tracking Type Pet" under Alarm Settings (Beta Feature)
- Add "Tracking Schedule" Funktion und Alarm Settings/Tracking Schedule
- Add "Time to Stop Tracking" -> Object Stops/Object Disappears under Alarm Settings/Tracking Schedule
- Add "Pet Sensivity" under Alarm Settings/Smart Detection (Beta Feature)
- Add "Pet Alarm Delay" under Alarm Settings/Alarm Delay (Beta Feature)
- Split "Audio and Light" menue to separate menues Audio/Light
- Add dropdown under Light/Spotlight mode (off,Night smart mode/Timer Mode)
- Add "Pet" on Surveillance/Record Schedule (Beta Feature)
- Add "Pet" on Surveillance/Email Schedule (Beta Feature)
- Add "Pet" on Surveillance/Ftp Schedule (Beta Feature)
- Add "Pet" on Surveillance/Siren Schedule (Beta Feature)
- Add "Pet" on Surveillance/Push Schedule (Beta Feature)
- Rename "Network Setting" to "Network"
- Add "Wi-Fi Channel Prefer" under Network/Saved Wi-Fi network (Auto/Only 2.4G/5G)
- Rename "Port Settings" to "Server Settings" under Network
- Rename "Media Port" to Basic Service under Network/Server Settings
- Add "Illegal Login" Lockout to System/User Management
- Bugfix TimeZone can now be changed under System/Date & Time

Regards Chris

**Comment by AT0myks** [comment]
Where in the web interface can you see the changelog?

**Comment by combatlord** [comment]
There are no changelog in WebGui.
I mean the changelist refers to Webgui and not the PC-Client. 
Ok you are right, my mistake. ;-)

I have two cams. 

One of them with FW 1643 and the other with the newest 2217. 
I has only compare the changes to the previous version.

The FW 1643 was the Version when i bought the Cam on Amazon two weeks ago.
I have ask the support to provide the 1643 but they said that can only found the 1584 which has some of the feature in this latest version.

**Reply by AT0myks** [reply]
Ok so if I understand correctly you manually compared the settings available in the web interface of both of your cameras and made a list of differences?

**Comment by combatlord** [comment]
Yes, that is exactly what i did.

**Reply by AT0myks** [reply]
Thank you for taking the time to make that list and sharing this firmware. Added in 2eadaf4968932dfe249d49ff580d303068f84ea6.
